### ISSUE TITLE
Airflow Flower Route error "Application is not available"

### ISSUE THREAD

**Issue by dmarcus-wire** [opendatahub-io-contrib/airflow-on-openshift#3]
The application is currently not serving requests at this endpoint. It may not have been started or is still starting.

Post deployment, I see the route, but I do not see any other flower resources or services that are being exposed to route. In other deployments, I see the configs for the flower service defined in the airflow-values.yaml (https://github.com/redhat-na-ssa/mlops-prototype/blob/main/airflow/airflow-setup/airflow-values.yaml)

**Comment by codekow** [comment]
The example, cut and paste, does not deploy flower.

@dmarcus-wire Should we include flower with the example deployment?

**Comment by shalberd** [comment]
@codekow @dmarcus-wire I did enable flower, yes. I also disabled the postgres that came with the helm chart, that image is totally mucked up. Using Crunchy instead, why reinvent the wheel.

However, routes in my case were never created by the helm charts, at least I did see either a flower nor an Airflow route. No wonder, they are not present in Kubernetes, they have ingress only.

i.e. for the airflow web GUI, I had to create it post-helm deployment.

```
kind: Route
apiVersion: route.openshift.io/v1
metadata:
  name: airflow-webserver
  namespace: airflow
spec:
  to:
    kind: Service
    name: airflow-webserver
    weight: 100
  port:
    targetPort: airflow-ui
  tls:
    termination: edge
    insecureEdgeTerminationPolicy: Redirect
  wildcardPolicy: None
```

https://github.com/opendatahub-io-contrib/airflow-on-openshift/issues/7#issuecomment-1599585068

Summing it up, not using postgres that came with the helm charts helped a lot.
Plus, some coordination with thesuperzapper from airflow helm central, who moved from Australia and had been busy, but in the end helped me out a whole lot.

**Comment by shalberd** [comment]
@dmarcus-wire can we connect on open data hub Slack, please? I feel a lot has changed since April, for the better.
Keyword: Stackable Operator for Airflow, Red Hat Certified Base Images and so on.

Now, Stackable requires an external Postgres, too, i.e. via Crunchy Postgres Operator or an enterprise install outside Openshift, but still, in my view, this is a real game-changer.

https://odh-io.slack.com/archives/C033X6XM448/p1691415742481449?thread_ts=1688000061.441579&cid=C033X6XM448

**Comment by codekow** [comment]
Issue addressed in PR #9
